src/main/java/io/jenkins/plugins/gitlabbranchsource/LogCommentTrait.java

```diff
@@ -0,0 +1,53 @@
+package io.jenkins.plugins.gitlabbranchsource;
+
+import hudson.Extension;
+import jenkins.scm.api.SCMSource;
+import jenkins.scm.api.trait.SCMSourceContext;
+import jenkins.scm.api.trait.SCMSourceTrait;
+import jenkins.scm.api.trait.SCMSourceTraitDescriptor;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+public class LogCommentTrait extends SCMSourceTrait {
```

Comment:
LogCommentTrait should have a jelly config and databoundsetter for sudo :)

Reply:
Fixed in 883df7f